src/filter_finder.py

```diff
@@ -0,0 +1,165 @@
+			skateboard_mask = cv2.bitwise_or(board_mask, wheel_mask)
+			skateboard_mask = cv2.bitwise_or(skateboard_mask, edge_mask)
+			
+			#Extract features from frame using masks
+			extracted_board = cv2.bitwise_and(frame, frame, mask = board_mask)
+			extracted_wheel = cv2.bitwise_and(frame, frame, mask = wheel_mask)
+			extracted_edge = cv2.bitwise_and(frame, frame, mask = edge_mask)
+			extracted_skateboard = cv2.bitwise_and(frame, frame, mask = skateboard_mask)
+			
+			#show raw frame
+			cv2.imshow('raw frame', frame)
+
+			#show all extracted features
+			cv2.imshow('extracted board', extracted_board)
+			cv2.imshow('extracted wheels', extracted_wheel)
+			cv2.imshow('extracted edges', extracted_edge)
+			cv2.imshow('extracted skateboard', extracted_skateboard)
+			
+			#exit early if escape key pressed
+			k = cv2.waitKey(5) & 0xFF
+			if k == 27:
+				break
+
+			#keyboard controls for altering the lower bounds of experimental color range
+			if k == ord('a'):
+				lower_chosen[0] = (lower_chosen[0] + sinc) % 256
+			if k == ord('s'):
+				lower_chosen[1] = (lower_chosen[1] + sinc) % 256
+			if k == ord('d'):
+				lower_chosen[2] = (lower_chosen[2] + sinc) % 256
+			if k == ord('z'):
+				lower_chosen[0] = (lower_chosen[0] - sinc) % 256
+			if k == ord('x'):
+				lower_chosen[1] = (lower_chosen[1] - sinc) % 256
+			if k == ord('c'):
+				lower_chosen[2] = (lower_chosen[2] - sinc) % 256
+				
+			if k == ord('A'):
+				lower_chosen[0] = (lower_chosen[0] + binc) % 256
+			if k == ord('S'):
+				lower_chosen[1] = (lower_chosen[1] + binc) % 256
+			if k == ord('D'):
+				lower_chosen[2] = (lower_chosen[2] + binc) % 256
+			if k == ord('Z'):
+				lower_chosen[0] = (lower_chosen[0] - binc) % 256
+			if k == ord('X'):
+				lower_chosen[1] = (lower_chosen[1] - binc) % 256
+			if k == ord('C'):
+				lower_chosen[2] = (lower_chosen[2] - binc) % 256
+
+			#keyboard controls for altering the upper bounds of experimental color range
+			if k == ord('g'):
+				upper_chosen[0] = (upper_chosen[0] + sinc) % 256
+			if k == ord('h'):
+				upper_chosen[1] = (upper_chosen[1] + sinc) % 256
+			if k == ord('j'):
+				upper_chosen[2] = (upper_chosen[2] + sinc) % 256
+			if k == ord('b'):
+				upper_chosen[0] = (upper_chosen[0] - sinc) % 256
+			if k == ord('n'):
+				upper_chosen[1] = (upper_chosen[1] - sinc) % 256
+			if k == ord('m'):
+				upper_chosen[2] = (upper_chosen[2] - sinc) % 256
+
+			if k == ord('G'):
+				upper_chosen[0] = (upper_chosen[0] + binc) % 256
+			if k == ord('H'):
+				upper_chosen[1] = (upper_chosen[1] + binc) % 256
+			if k == ord('J'):
+				upper_chosen[2] = (upper_chosen[2] + binc) % 256
+			if k == ord('B'):
+				upper_chosen[0] = (upper_chosen[0] - binc) % 256
+			if k == ord('N'):
+				upper_chosen[1] = (upper_chosen[1] - binc) % 256
+			if k == ord('M'):
+				upper_chosen[2] = (upper_chosen[2] - binc) % 256
+				
+			#keyboard controls to reset experimental color range
+			if k == ord('r'):
+				lower_chosen[0] = 0
```

Comment:
something like:
```suggestion
				lower_chosen[:] = 0
```
?